### PR TITLE
References updated. Added links and fixed formatting (#1105)

### PR DESCRIPTION
**Pull request by MarcoBarroca**
## Changes
I had to redo PR #1107  because git wasn't properly configured with my GitHub e-mail and I couldn't sign the CLA. Fixes the References section formatting mismatch in the Unsupervised learning notebook and adds a missing link in the Quantum Feature maps notebook in the Machine Learning Chapter. Issues #716 and #1105 

## Implementation details
Corrected the formatting of the references in the mentioned Notebooks

## How to read this PR
Check the differences in the reference section in the mentioned notebooks.

## Screenshots
Quantum feature maps notebook:
Screen Shot 2022-06-06 at 16 14 26

Unsupervised Learning Notebook:
Screen Shot 2022-06-06 at 16 12 28

### CODE REVIEW

File: notebooks/quantum-machine-learning/kernel.ipynb [hunk 1]
```diff
@@ -746,7 +746,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/kernel.ipynb:749]: Thanks for your PR @MarcoBarroca ! Could you remove these changes that are not related with the content? 🙏

File: notebooks/quantum-machine-learning/kernel.ipynb [hunk 2]
```diff
@@ -760,7 +760,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9"
+   "version": "3.10.4"
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/kernel.ipynb:763]: Same for this change.

File: notebooks/quantum-machine-learning/unsupervised.ipynb [hunk 3]
```diff
@@ -38,17 +38,17 @@
...
     "\n",
-    "3.  [Amin, M. H., Andriyash, E., Rolfe, J., Kulchytskyy, B., & Melko, R. (2018). Quantum boltzmann machine. Physical Review X, 8(2), 021050.](https://journals.aps.org/prx/abstract/10.1103/PhysRevX.8.021050)"
+    "3.  Amin, M. H., Andriyash, E., Rolfe, J., Kulchytskyy, B., & Melko, R. (2018). Quantum boltzmann machine. Physical Review X, 8(2), 021050., [doi.org:10.1103/PhysRevX.8.021050)](https://doi.org/10.1103/PhysRevX.8.021050), [arXiv:1601.02036](https://arxiv.org/abs/1601.02036)"
    ]
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/unsupervised.ipynb:51]: Same as the changes from my previous reviews.

File: notebooks/quantum-machine-learning/unsupervised.ipynb [hunk 4]
```diff
@@ -62,7 +62,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9"
+   "version": "3.10.4"
```
Comment (Tansito) [inline notebooks/quantum-machine-learning/unsupervised.ipynb:65]: The last one change to be removed!

**Review by Tansito — CHANGES_REQUESTED** [review]

File: notebooks/quantum-machine-learning/unsupervised.ipynb (hunk 4 above)
Comment (MarcoBarroca) [reply notebooks/quantum-machine-learning/unsupervised.ipynb]: I just removed those changes!

**Review by frankharkins — DISMISSED** [review]
Awesome, thank you!

**Comment by MarcoBarroca** [comment]
Hey @Tansito the change request still seems to be blocking the merge despite being resolved.

**Comment by Tansito** [comment]
Thanks @MarcoBarroca , I asked again to @frankharkins again for the review.

**Review by Tansito — APPROVED** [review]

**Pull request merged**